huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -320,6 +322,24 @@ func ResourceFgsFunctionV2() *schema.Resource {
 				Description: "The versions management of the function.",
 			},
 			"tags": common.TagsSchema(),
+			"qualifier_name": {
```

Comment:
The newly added Optional type fields all have return values. Can Computed be added?

Reply:
If this parameter is not configured when wearing the item resource, they have no value.